core/lib.rs

```diff
@@ -1,9 +1,9 @@
 // Copyright 2018-2022 the Deno authors. All rights reserved. MIT license.
 mod async_cancel;
 mod async_cell;
-mod bindings;
+pub mod bindings;
 pub mod error;
-mod error_codes;
+pub mod error_codes;
```

Comment:
These two are red flags for me; they shouldn't be public modules. What's the reason?

Reply:
Code generated ops call into some of these functions, the cleaner solution might be to group the functions ops call into a single mod

Reply:
Please expose only the functions that are actually being used (re-export from `lib.rs`) rather than making both of these modules completely public.